tomcat/manifest.json

```diff
@@ -31,11 +31,14 @@
       "spec": "assets/configuration/spec.yaml"
     },
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "tomcat--overview": "assets/dashboards/overview.json",
```

Comment:
why two `-`?
Why are there two dashboards?

Reply:
This is the old short name for the dashboard in dogweb. There were two dashboards in dogweb, so we are going to migrate together